spec/lib/data_type_spec.rb

```diff
@@ -49,6 +49,18 @@
     it { is_expected.to eq 'MyInteger' }
   end
 
+  describe 'Types in array with inherited entity_name' do
```

Comment:
mmh, thin [reference_entity_spec.rb](https://github.com/ruby-grape/grape-swagger/blob/master/spec/swagger_v2/reference_entity_spec.rb) will be a better place for it

Reply:
I copied and modified the test that is above this one in this spec. As this is testing the unit of handling calculating the entity.  I can add tests there as well.